versions/3.1.0.md

```diff
@@ -1943,7 +1945,7 @@ Field Name | Type | Description
 <a name="exampleSummary"></a>summary | `string` | Short description for the example.
 <a name="exampleDescription"></a>description | `string` | Long description for the example. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
 <a name="exampleValue"></a>value | Any | Embedded literal example. The `value` field and `externalValue` field are mutually exclusive. To represent examples of media types that cannot naturally represented in JSON or YAML, use a string value to contain the example, escaping where necessary.
-<a name="exampleExternalValue"></a>externalValue | `string` | A URL that points to the literal example. This provides the capability to reference examples that cannot easily be included in JSON or YAML documents.  The `value` field and `externalValue` field are mutually exclusive. 
+<a name="exampleExternalValue"></a>externalValue | `string` | A URI that points to the literal example. This provides the capability to reference examples that cannot easily be included in JSON or YAML documents.  The `value` field and `externalValue` field are mutually exclusive.
```

Comment:
How would this work with a URI that is not a URL?  A consumer would not know how to resolve the URI to locate the external example.

Reply:
Then the user would have to make the resource known to the implementation via some other means. This is what JSON Schema specifies.  https://tools.ietf.org/html/draft-bhutton-json-schema-00#section-9.1.2

Reply:
Yes as @karenetheridge says this is for consistency with JSON Schema's `$ref`, and as I amended it, our Reference Object.